### ISSUE TITLE
vCheck Schedule Task Not Working

### ISSUE THREAD

**Issue by captainaa4** [alanrenouf/vCheck-vSphere#635]
This is in reference of thread vCheck Not working  #363
==========================
Hi Sneddo/Alan
 I want to re-open this thread because I am facing same kind of issue. I am using vCheck ver 6.23 and I have created 10 Schedule tasks for my 10 vCenters. Out of 10 only 6 tasks are working fine and I am getting report on mail but 4 Tasks are not stopping and keeps running for 2 days.
 Also when I try to run the task manually also it did not stop but when I run the script manually and it works for all of the vCenters.
 All tasks are configured for below configuration. Kindly suggest further asap.

Program/script: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
Arguments: -file vcheck.ps1 -OutputPath C:\scripts\vCheck_Scripts\Report
Start in: C:\scripts\vCheck_Scripts\

**Comment by Sneddo** [comment]
Are you running the scheduled task as the same user that you ran vCheck as manually? Most of the time this will highlight the issue as the script will pause at the same place...

**Comment by captainaa4** [comment]
Yes, I am using same credentials in Schedule task which I use to run manually. Also I tried to run the schedule task from another server which was not working but could not get any luck. Currently I am running these scripts manually on daily basis for my vCenters which is not running by schedule task.
Please help......

**Comment by mrclschstr** [comment]
I'm experiencing those issues lately as well. I run several vCheck jobs (which means all scripts are the **same**) under the same user and one of those jobs doesn't finish. When I run this job manually with the same service account it works like a charm. As there are no meaningful error messages and the vCheck stops at different scripts every run, I really cannot catch the issue. Any help appreciated.

My environment:
- vCheck v6.25
- Powershell 5.1.14409
- PowerCLI 6.5.4.7
- vCenter 5.5

**Comment by a13xchan** [comment]
No issues here running 13 vChecks, different accounts logging into each vCenter in difference domains. I also have the script running as a scheduled task under Windows 2016 (previously 2008 and 2012) as the different service users. vCheck is the same apart from global settings and vcenter connection settings

The task auto stops if it runs more than 2hrs. vCheck 6.25, vCenter 6 and vCenter 6.5, latest version of PowerCLI.

If you can run it manually it will run as a scheduled task . Are you running it as a domain account and are there any restrictions on your domain to limit the account you are using from logging in. Are you running any other powershell sessions and have you set a limit?

Try change the time your jobs run so if you have 10, run 5 at one time and then run another 5 after the others have finished.

**Comment by Scepticalist** [comment]
See my reply on issue #642 - I had specific issues running vCheck on a non-vCenter server with the login process and outputting the vCheck script helped to diagnose it easily.
